docs/tutorials/mastering.md

```diff
@@ -847,38 +850,42 @@ Log out of the VM.
 
 ### Prepare the Juju Controller VM
 
-Log in to the `juju-controller` VM.
+Log in to the `juju-controller` VM:
 
 ```console
 multipass shell juju-controller
 ```
 
-Begin by installing MicroK8s to hold the Juju controller. Configure MetalLB to expose one IP address for the controller (`10.201.0.50`) and one for the Canonical Observability Stack (`10.201.0.51)`.
+Begin by installing MicroK8s to hold the Juju controller. 
+Configure MetalLB to expose one IP address for the controller (`10.201.0.50`) and one for the Canonical Observability Stack (`10.201.0.51)`:
 
 ```console
-sudo snap install microk8s --channel=1.27-strict/stable
+sudo snap install microk8s --channel=1.29-strict/stable
 sudo microk8s enable hostpath-storage
 sudo microk8s enable metallb:10.201.0.50-10.201.0.51
 sudo usermod -a -G snap_microk8s $USER
 newgrp snap_microk8s
 ```
 
-Update MicroK8s DNS to point to our DNS server
+Update MicroK8s DNS to point to our DNS server:
 
 ```console
 sudo microk8s disable dns
 sudo microk8s enable dns:10.201.0.100
 ```
 
-Install Juju and bootstrap the controller to the local Microk8s install as a load balancer service. This will expose the Juju controller on the first allocated MetalLB address.
+Install Juju and bootstrap the controller to the local MicroK8s install as a LoadBalancer service.
+This will expose the Juju controller on the first allocated MetalLB address:
 
 ```console
 mkdir -p ~/.local/share/juju
 sudo snap install juju --channel=3.1/stable
 juju bootstrap microk8s --config controller-service-type=loadbalancer sdcore
 ```
 
-At this point, the Juju controller is ready to start managing external clouds. Add the Kubernetes clusters representing the user plane, control plane, and gNB simulator to Juju. This is done by using the Kubernetes configuration file generated when setting up the clusters above.
+At this point, the Juju controller is ready to start managing external clouds. 
```

Comment:
Having all these line breaks added makes for a messy review. Can we not make changes to whitespace only to make this easier to spot real changes?

Reply:
Sorry 'bout that. 
Since it's already there, I'd rather not revert it, but I'll keep that in mind for the next time.